packages/flutter/lib/src/material/drawer_header.dart

```diff
@@ -51,20 +91,93 @@ class DrawerHeader extends StatelessWidget {
           )
         )
       ),
-      child: new Stack(
-        children: <Widget>[
-          background ?? new Container(),
-          new Positioned(
-            top: statusBarHeight + 16.0,
-            left: 16.0,
-            right: 16.0,
-            bottom: 8.0,
-            child: new DefaultTextStyle(
-              style: Theme.of(context).textTheme.body2,
-              child: content
+      child: new Container(
+        padding: const EdgeInsets.only(left: 16.0, right: 16.0, bottom: 8.0),
+        decoration: config.decoration ?? new BoxDecoration(),
+        child: new Column(
+          children: <Widget>[
+            new Flexible(
+              child: new Stack(
+                children: <Widget>[
+                  new Positioned(
+                    top: statusBarHeight + 16.0,
+                    right: 0.0,
+                    child: new Row(
+                      children: config.otherAccountsPictures.map(
```

Comment:
Shouldn't it be `config.otherAccountsPictures?.map`? Also, can `children` be `null` inside of a `Row`?